### ISSUE TITLE
how  optimize a noise image with tensorflow?

### ISSUE THREAD

**Issue by molo32** [tensorflow/tensorflow#45284]
how to apply sgd to an image with tensorflow?

I have these elements:
* image noise
* target image
* optimizer
* loss function
Is there a way to optimize a noise image so that it looks more like a target image?
first it starts with noise then we calculate the difference or error between image noise and target image, then we calculate the gradient and apply it to the noise image.

**Comment by Saduf2019** [comment]
@molo32 
Please use [Denoising AutoEncoder](https://www.pyimagesearch.com/2020/02/24/denoising-autoencoders-with-keras-tensorflow-and-deep-learning/), and move this ticket to closed status as it is not a bug or feature request, you may open a ticket on stackoverflow for any further help on this.
